CHANGES.md

```diff
@@ -64,6 +64,9 @@
 - The test suite for Coq now requires Coq >= 8.16 due to changes in the
   plugin loading mechanism upstream (which now uses findlib).
 
+- on macOS, sign executables produced by artifact substitution (#6137, fixes
```

Comment:
`On MacOS`.

Reply:
I believe it should be `On macOS` since that is how Apple styles it: https://www.apple.com/macos/monterey/